### PR TITLE
Add Timeseries Notebook Plots

### PR DESCRIPTION
**Pull request by zackAemmer**
- Creates a notebook called "timeseries_analysis" that contains plots over time for various aspects of the OpenPATH data
- Follows structure of other plotting notebooks for gathering data from the db, and having user enter inputs in the first cell
- Adds entries to the frontend list which allow the plots to be loaded into the public dashboard

### CODE REVIEW

File: viz_scripts/timeseries_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Count the number of trips for each sensed mode\n",
```
Comment (shankari) [inline viz_scripts/timeseries_analysis.ipynb:220]: nit: `confirmed mode`, not `sensed mode` in the comment

File: viz_scripts/timeseries_analysis.ipynb [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot of ebike mode share across many individual users\n",
+    "# This plot is a little crazy if the number of users displayed is not limited\n",
```
Comment (shankari) [inline viz_scripts/timeseries_analysis.ipynb:434]: Yeah that is fine, most programs have < 50 users each.
Can you add an example of what this looks like for the first 50 users, for example?
we can just leave it off the metrics list if it looks too wonky.

**Review by shankari — CHANGES_REQUESTED** [review]
@zackAemmer This looks great! Well written and very clean to understand.
The one additional change that you will need is to add an entry into the crontab to run the notebook and generate the metrics.

https://github.com/e-mission/em-public-dashboard/blob/main/viz_scripts/docker/crontab

Also, in terms of GitHub best practices, you might want to submit PRs from a branch in the future to avoid messing up your main branch in case you need to revert changes, or we end up with a squash-merge.

File: viz_scripts/timeseries_analysis.ipynb (hunk 2 above)
Comment (zackAemmer) [reply viz_scripts/timeseries_analysis.ipynb:434]: Here is an example with the first 20 user IDs:

ts_ebike_mile_individual_prepilot

File: viz_scripts/timeseries_analysis.ipynb (hunk 1 above)
Comment (zackAemmer) [reply viz_scripts/timeseries_analysis.ipynb]: Switched to confirmed mode

File: viz_scripts/timeseries_analysis.ipynb (hunk 2 above)
Comment (shankari) [reply viz_scripts/timeseries_analysis.ipynb:425]: hm that is not as pretty as I had hoped. I wonder if a monthly bar graph would be more readable than a daily graph.
I will go ahead and publish this now, but if you are looking for something to do, trying to split by month and/or show as a bar graph may be more readable.

Comment (shankari) [reply viz_scripts/timeseries_analysis.ipynb:425]: or even weekly

And maybe a second one for trip count instead of distance.

**Review by shankari — APPROVED** [review]
Assuming it all works, LGTM!

**Pull request merged**